develop/buf-breaking.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
```

Comment:
It might be helpful to add a short comment at the top to summarize what the script is for. It took me a while to wrap me head around what was going on here.